docs/modeling/models.rst

```diff
@@ -252,6 +252,20 @@ though.
     assign custom inverses at your own risk.
 
 
+Model Separability
+==================
+
+Simple models have a boolean `Model.separable <astropy.modeling.Model.separable>` property.
+It indicates whether the outputs are independent and is essential for computing the
+separability of compound models using the :func:`~astropy.modeling.is_separable` function.
+Having a separable compound model means that it can be decomposed into independent models,
+whic in turn is useful in many applications.
+Sometimes it is easier to define inverses using the independent parts than the
```

Comment:
I am not sure how this sentence relates to "separability," please clarify.

Reply:
I hope it's clearer now but let me know if not.